account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -0,0 +1,821 @@
+                parsed_subtotal = parsed_inv['lines'][i]['price_subtotal']
+                if float_compare(
+                        odoo_subtotal, parsed_subtotal,
+                        precision_rounding=prec):
+                    diff_amount = float_round(
+                        parsed_subtotal - odoo_subtotal,
+                        precision_rounding=prec)
+                    logger.info(
+                        'Price subtotal difference found on invoice line %d '
+                        '(source:%s, odoo:%s, diff:%s).',
+                        i + 1, parsed_subtotal, odoo_subtotal, diff_amount)
+                    copy_dict = {
+                        'name': _('Adjustment on %s') % iline.name,
+                        'quantity': 1,
+                        'price_unit': diff_amount,
+                    }
+                    if import_config['invoice_line_method'] ==\
+                            'nline_auto_product':
+                        copy_dict['product_id'] = False
+                    # Add the adjustment line
+                    iline.copy(copy_dict)
+                    logger.info('Adjustment invoice line created')
+        if float_compare(
+                parsed_inv['amount_untaxed'], invoice.amount_untaxed,
+                precision_rounding=prec):
+            # create global ajustment line
+            diff_amount = float_round(
+                parsed_inv['amount_untaxed'] - invoice.amount_untaxed,
+                precision_rounding=prec)
+            logger.info(
+                'Amount untaxed difference found '
+                '(source: %s, odoo:%s, diff:%s)',
+                parsed_inv['amount_untaxed'], invoice.amount_untaxed,
+                diff_amount)
+            il_vals = self._prepare_global_adjustment_line(
+                diff_amount, invoice, import_config)
+            il_vals['invoice_id'] = invoice.id
+            self.env['account.invoice.line'].create(il_vals)
+            logger.info('Global adjustment invoice line created')
+        # Invalidate cache
+        invoice = self.env['account.invoice'].browse(invoice.id)
+        assert not float_compare(
+            parsed_inv['amount_untaxed'], invoice.amount_untaxed,
+            precision_rounding=prec)
+        # Force tax amount if necessary
+        if float_compare(
+                invoice.amount_total, parsed_inv['amount_total'],
+                precision_rounding=prec):
+            if not invoice.tax_line_ids:
+                raise UserError(_(
+                    "The total amount is different from the untaxed amount, "
+                    "but no tax has been configured !"))
+            initial_tax_amount = invoice.tax_line_ids[0].amount
+            tax_amount = parsed_inv['amount_total'] -\
+                parsed_inv['amount_untaxed']
+            invoice.tax_line_ids[0].amount = tax_amount
+            cur_symbol = invoice.currency_id.symbol
+            invoice.message_post(
```

Comment:
Could you make the message translatable, as it was done in V10? (the fix of PR https://github.com/OCA/edi/pull/51 should be ported to V11)

Reply:
done